kits/README.md

```diff
@@ -6,15 +6,15 @@ In each starter kit folder we give you all the tools necessary to compete. Make
 
 To run a episode with verbosity level 2 (higher is more verbose), seed 42, and save a replay to replay.json:
 
-```
+```Python console
 luxai-s2 kits/python/main.py kits/python/main.py -s 42 -v 2 -o replay.json
 ```
 
 To then watch the replay, upload replay.json to http://s2vis.lux-ai.org/
 
 Alternatively you can generate a openable HTML file to watch it as well by specifying the output as .html
 
-```
+```Python console
```

Comment:
these should be called `python` not `Python console` I believe.

Reply:
I can certainly change this, but I don't think there is a difference between `python` and `python console` highlighting for this specific block of code